src/server/pkg/deploy/cmds/cmds.go

```diff
@@ -284,3 +292,18 @@ you used the "--static-etcd-volume" flag), the underlying volume will not be
 removed.`)
 	return []*cobra.Command{deploy, undeploy}
 }
+
+func getDefaultDashImage() string {
+	// Try to get the latest according to our pachctl version, or fall back to the encoded default
+	url := fmt.Sprintf("https://pachyderm.io/versions/dash/%v.%v/latest", version.MajorVersion, version.MinorVersion)
```

Comment:
I disagree, but perhaps our preferences or goals are different.

I want a tool that stays out of my way in the most commonly used cases (in this case, upgrading). And I want a tool that is easy for our less technical users (who are the ones that need the UI the most) to use.

I think it's a pretty common practice to host something as part of an install. I see rvm, brew, gsutil, and gcloud all have one liners that curl a thing and pipe it to bash. I'm pretty certain I've seen the URL-just-to-get-the-latest version pattern as well.

We could provide the one-liner you suggest in the docs which would be easy for anyone to copy/paste. However, if we find ourselves using (and recommending) this method the only real difference is putting it in the code or making it a convention (having it in the docs).

I argue that having it in the docs is less convenient to novice users, less convenient for me, and is no more robust than having it in the code.

I get how this couples us to another service (github.io). I don't think the site being down is a common enough case for us to worry much about it. In this case the code defaults to the value hardcoded. If we provide the one liner via the docs (and that's the recommended method for users), it doesn't make it work better if github.io is down. In fact it'd probably return a confusing error to the user.

Anyway. That's my argument for the feature as-is. The choice is document the one liner in the docs or keep it in the code

---

OR

I realized we have a third option. If I update the release process on the UI side to always additionally tag and push an image like `pachyderm/dash:1.3-latest` (and we only update `pachyderm/dash:1.3-stable` when doing a point release) then we can just keep the flag (for debugging) and avoid the URLs altogether. 

I like that best because then I don't have to upgrade the release script to be smart and try to update those hardcoded image values. And that is a really sane and easy default for novice users, both for a standard deployment and upgrading their UI.